### PR TITLE
fix: (2.34) Fix NPE by adding new request parameter

### PR DESCRIPTION
**Pull request by huzvak**
DHIS2-7994

### CODE REVIEW

File: dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/EventController.java [hunk 1]
```diff
@@ -67,11 +65,7 @@
 import org.hisp.dhis.event.EventStatus;
 import org.hisp.dhis.fieldfilter.FieldFilterParams;
 import org.hisp.dhis.fieldfilter.FieldFilterService;
-import org.hisp.dhis.fileresource.FileResource;
-import org.hisp.dhis.fileresource.FileResourceDomain;
-import org.hisp.dhis.fileresource.FileResourceService;
-import org.hisp.dhis.fileresource.FileResourceStorageStatus;
-import org.hisp.dhis.fileresource.ImageFileDimension;
+import org.hisp.dhis.fileresource.*;
```
Comment (mortenoh) [inline dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/EventController.java]: don't collapse imports

File: dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/EventController.java [hunk 2]
```diff
@@ -98,28 +92,12 @@
...
-import java.io.OutputStream;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.zip.GZIPOutputStream;
+import org.springframework.web.bind.annotation.*;
```
Comment (mortenoh) [inline dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/EventController.java]: don't collapse imports

Comment (maikelarabori) [reply dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/EventController.java]: Ok, I will fix my IDE and avoid it

Comment (stian-sandvold) [reply dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/EventController.java]: If I remember correctly, we do collapse if more than 5 from same @mortenoh ?

File: dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/EventController.java (hunk 1 above)
Comment (huzvak) [reply dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/EventController.java]: I use the official style template. So if we do not want to collapse them, then we should change it there and let know the people to update.

But I have also turned off automatic optimization of imports on commit.

**Comment by vilkg** [comment]
Functionally works fine on all the back-ported versions. Please move the issue to done when this is fixed :) https://jira.dhis2.org/browse/DHIS2-7994

**Pull request merged**